definitions/src/generate_asm_constants.rs

```diff
@@ -110,7 +110,7 @@ fn main() {
         size_of::<AsmCoreMachine>()
     );
 
-    let m: Box<AsmCoreMachine> = Box::<AsmCoreMachine>::default();
+    let m: Box<AsmCoreMachine> = AsmCoreMachine::new(0, 0, 0);
```

Comment:
From this invocation, it is very easy to pass parameters in the wrong positions. Maybe it requires a builder as well.

Reply:
There are not many parameters at present, if there are more than 5 parameters, then it is reasonable to add a builder like `AsmCoreMachineBuilder`